src/ethereum_clis/cli_types.py

```diff
@@ -212,6 +213,23 @@ class TransitionToolInput(CamelModel):
     env: Environment
 
 
+class TransitionToolConfig(CamelModel):
+    """Transition tool config."""
+
+    fork: Fork = Field(..., alias="network")
+    chain_id: int = Field(..., alias="chainid")
+    blob_schedule: BlobSchedule
```

Comment:
As discussed in chat group, let's pass instead:
```suggestion
    blob_params: BlobParams
```

Also, we already have a definition that fits our needs in here: https://github.com/ethereum/execution-spec-tests/blob/56ea9f2fdb157630101ecced198afae87752eae0/src/ethereum_test_base_types/composite_types.py#L487-L492

We could even rename that type to `BlobParams` because the current name `ForkBlobSchedule` doesn't make much sense.

Reply:
Agreed that the `ForkBlobSchedule` should be done on a separate as it touches a lot of files.